### PR TITLE
Add qualification documentation 

### PR DESCRIPTION
**Pull request by paultranvan**
This details how to qualify document with cozy-client by using the cozy-client [Qualification API](https://github.com/cozy/cozy-client/blob/master/docs/api/cozy-client.md#Qualification)

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -255,4 +258,4 @@ extra:
     repo: https://github.com/cozy/cozy-coclyco.git
     subdir: .
   search:
-    tokenizer: "[^a-z\u0430-\u044F\u04510-9\\-\\._]"
+    tokenizer: "[^a-z\u0430-\u044F\u04510-9\\-\\.]"
```
Comment (ptbrowne) [inline mkdocs.yml:261]: This change should be in another commit and should be documented.

File: src/tutorials/data/qualification.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+Note `saveFileQualification` is a cozy-client helper that can be replaced by:
+
+```js
+const qualifiedFile = Document.setQualification(file, qualification)
+await client
+    .collection('io.cozy.files')
+    .updateMetadataAttribute(file._id, qualifiedFile.metadata)
+
+```
```
Comment (ptbrowne) [inline src/tutorials/data/qualification.md:45]: I think we can removed this, this can drift quickly and am not sure that the implementation is relevant here ?

File: src/tutorials/data/qualification.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# Document qualification
+
+A document can be qualified by adding some attributes in its metadata meant to semantically describe the document. The objective is to maintain a transversal qualification structure between doctypes, so that files, banks, notes and so on could be qualified and retrieved. Currently, only the `io.cozy.files` doctype is supported.
```
Comment (ptbrowne) [inline src/tutorials/data/qualification.md:3]: d/some/

**Review by ptbrowne — APPROVED** [review]
Thanks

File: mkdocs.yml (hunk 1 above)
Comment (paultranvan) [reply mkdocs.yml:261]: Not sure where this comes from, is it because the mkdocs.yml was not up to date?

Comment (ptbrowne) [reply mkdocs.yml:261]: Ah sure, you're right. I was surprised to see changes to the mkdocs.yml but I understand now : Iit's not really a problem if it's not up-to-date since it's a generated file. It could be removed from the repository, I think it's a bit misleading to have it here.

Comment (paultranvan) [reply mkdocs.yml:261]: Ah ok, so we should add it to .gitignore?

File: src/tutorials/data/qualification.md (hunk 2 above)
Comment (paultranvan) [reply src/tutorials/data/qualification.md:45]: I assumed it could be useful to have it here, but no problem to remove

File: mkdocs.yml (hunk 1 above)
Comment (paultranvan) [reply mkdocs.yml]: Done with 930822b3fa4f09cebe4bb3bfe7373fbcc5aa0d67

**Comment by ptbrowne** [comment]
LGTM, thanks
